test/unit/user_test.rb

```diff
@@ -26,6 +26,11 @@ class UserTest < ActiveSupport::TestCase
     assert_equal user.notes, user.drupal_user.notes
   end
 
+  test "user questions" do
+    user = rusers(:jeff)
+    assert user.questions.count.count > 0
```

Comment:
I've found a problem here. the `questions` method is returning a `DrupalNode::FriendlyIdActiveRecordRelation`, so when i call `.count` it returns an `ActiveSupport::OrderedHash`, and it gives an error when I try to compare it with the number `0`.

How should solve this problem?